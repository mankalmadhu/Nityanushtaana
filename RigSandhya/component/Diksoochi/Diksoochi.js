import React from 'react';
import {Navigation, Card} from 'react-router-navigation';

import Parividi from '../Parividi/Parividi';
import Shreekara from '../Shreekara/Shreekara';
import Shershike from '../Shershike/Shershike';

export default Diksoochi = () => (
      <Navigation navBarStyle={{backgroundColor: '#C1C1C1'}}>
        <Card exact path="/" component={Parividi} renderTitle = {()=>(<Shershike shershike='ಋಗ್ವೇದ ಸ೦ಧ್ಯಾವ೦ದನ'/>)}/>
        <Card path="/ಪ್ರಾತಸ್ಸ೦ಧ್ಯಾ"  component={Shreekara} renderTitle = {()=>(<Shershike shershike='ಪ್ರಾತಸ್ಸ೦ಧ್ಯಾ'/>)} />
        <Card path="/ಪ್ರಾತಸ್ಸಮಿಧಾದಾನ"  component={Shreekara} renderTitle = {()=>(<Shershike shershike='ಪ್ರಾತಸ್ಸಮಿಧಾದಾನ'/>)} />
        <Card path="/ಮಾಧ್ಯಾಹ್ನಿಕಸ೦ಧ್ಯಾ"  component={Shreekara} renderTitle = {()=>(<Shershike shershike='ಮಾಧ್ಯಾಹ್ನಿಕಸ೦ಧ್ಯಾ'/>)} />
        <Card path="/ಪ್ರಾತಸ್ಸಮಿಧಾದಾನ"  component={Shreekara} renderTitle = {()=>(<Shershike shershike='ಪ್ರಾತಸ್ಸಮಿಧಾದಾನ'/>)} />
        <Card path="/ಸಾಯ೦ಸ೦ಧ್ಯಾ"  component={Shreekara} renderTitle = {()=>(<Shershike shershike='ಸಾಯ೦ಸ೦ಧ್ಯಾ'/>)} />
        <Card path="/ಸಾಯ೦ಸಮಿಧಾದಾನ"  component={Shreekara} renderTitle = {()=>(<Shershike shershike='ಸಾಯ೦ಸಮಿಧಾದಾನ'/>)} />
        <Card path="/ಯಜ್ಞೋಪವೀತ"  component={Shreekara} renderTitle = {()=>(<Shershike shershike='ಯಜ್ಞೋಪವೀತ'/>)} />
        <Card path="/ಉಪನಯನಕಾರಿಕ"  component={Shreekara} renderTitle = {()=>(<Shershike shershike='ಉಪನಯನಕಾರಿಕ'/>)} />
      </Navigation>
     
  );
  