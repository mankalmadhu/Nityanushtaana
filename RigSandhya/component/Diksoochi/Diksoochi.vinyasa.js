import { StyleSheet } from 'react-native';

export default dikSoochiVinyasa = StyleSheet.create({
  shershikeAvarana: {
    backgroundColor: '#595959',
    marginTop: 20
  }
  
});
 