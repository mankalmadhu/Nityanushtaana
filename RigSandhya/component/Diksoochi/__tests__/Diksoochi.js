import React from 'react';
import Renderer from 'react-test-renderer';
import Diksoochi from '../Diksoochi';

jest.mock('../../Shershike/Shershike', ()=> 'Shershike');
jest.mock('../../Shreekara/Shreekara', ()=> 'Shreekara');
jest.mock('../../Parividi/Parividi', ()=> 'Parividi');
jest.mock('../../PratahSandhya/PratahSandhya', ()=> 'ಪ್ರಾತಸ್ಸ೦ಧ್ಯಾ');
jest.mock('../../PratahSamidhaDana/PratahSamidhaDana', ()=> 'ಪ್ರಾತಸ್ಸಮಿಧಾದಾನ');
jest.mock('../../MadhyahnikaSandhya/MadhyahnikaSandhya', ()=> 'ಮಾಧ್ಯಾಹ್ನಿಕಸ೦ಧ್ಯಾ');
jest.mock('../../SayamSandhya/SayamSandhya', ()=> 'ಸಾಯ೦ಸ೦ಧ್ಯಾ');
jest.mock('../../SayamSamidhaDana/SayamSamidhaDana', ()=> 'ಸಾಯ೦ಸಮಿಧಾದಾನ');
jest.mock('../../UpanayanaKarika/UpanayanaKarika', ()=> 'ಉಪನಯನಕಾರಿಕಾ');

jest.mock('react-router-navigation', (props)=>{
  const MockReact = require('React');
  class Navigation extends MockReact.Component
  {
    render() {
      return MockReact.createElement('Navigation' , this.props);
    }
  }

  class Card extends MockReact.Component
  {
    render() {
      return MockReact.createElement('Card' ,this.props);
    }
  }

  return ({Navigation : Navigation,
          Card : Card });
}) ;

  
test("Diksoochi" , () => {
  const diksoochi = Renderer.create(
    <Diksoochi/>
   ).toJSON();

  expect(diksoochi).toMatchSnapshot();
  
});
  