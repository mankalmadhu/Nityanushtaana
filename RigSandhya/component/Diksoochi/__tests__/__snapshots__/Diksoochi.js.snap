// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Diksoochi 1`] = `
<Navigation
  navBarStyle={
    Object {
      "backgroundColor": "#C1C1C1",
    }
  }
>
  <Card
    component={[Function]}
    exact={true}
    path="/"
    renderTitle={[Function]}
  />
  <Card
    component={[Function]}
    path="/ಪ್ರಾತಸ್ಸ೦ಧ್ಯಾ"
    renderTitle={[Function]}
  />
  <Card
    component={[Function]}
    path="/ಪ್ರಾತಸ್ಸಮಿಧಾದಾನ"
    renderTitle={[Function]}
  />
  <Card
    component={[Function]}
    path="/ಮಾಧ್ಯಾಹ್ನಿಕಸ೦ಧ್ಯಾ"
    renderTitle={[Function]}
  />
  <Card
    component={[Function]}
    path="/ಪ್ರಾತಸ್ಸಮಿಧಾದಾನ"
    renderTitle={[Function]}
  />
  <Card
    component={[Function]}
    path="/ಸಾಯ೦ಸ೦ಧ್ಯಾ"
    renderTitle={[Function]}
  />
  <Card
    component={[Function]}
    path="/ಸಾಯ೦ಸಮಿಧಾದಾನ"
    renderTitle={[Function]}
  />
  <Card
    component={[Function]}
    path="/ಯಜ್ಞೋಪವೀತ"
    renderTitle={[Function]}
  />
  <Card
    component={[Function]}
    path="/ಉಪನಯನಕಾರಿಕ"
    renderTitle={[Function]}
  />
</Navigation>
`;
